Mainly styles fixes and some improvements to series single and landing pages

rocaiguina opened the pull request.

rocaiguina committed:
Fixes #12

rocaiguina committed:
Added internationalization files to new templates

rocaiguina committed:
add internationalization to content-series tempalte resolves #17

rocaiguina committed:
add subtitle to new single template resolves #20

rocaiguina committed:
This closes #25, closes #23 and fixes line spacing on p

rocaiguina committed:
This closes #13  ‘on landing page we should hide abstract section if content not available’

rocaiguina committed:
Closes #14 ‘handle last story on single page's next story component’

rocaiguina committed:
Closes #15 better handle title author on landing page

rocaiguina committed:
Closes #15 better handle title author on landing page and improves content boxes on landing page

rocaiguina committed:
Closes #18 improve mobile styles

rocaiguina committed:
Closes #24 fix author bio styles in regular posts

benlk commented on wp-content/themes/cpipr/series-landing.php:
```diff
@@ -174,7 +171,7 @@
 								'term' => $series,
 								'order' => 'DESC',
-								'posts_per_page' => $opt['per_page'],
+								'posts_per_page' => -1,
```
I can't find the link, but using `-1` to display all posts in a query is less performant than using a sufficiently large number, like `100`.

This setting should be changed through the series landing page settings, not hardcoded in the template, but if it's going to be changed here, the reason should be documented in a comment in the code, and it should still be a (large) fixed number.

rocaiguina replied:
ok, rolled back to original query

benlk commented on wp-content/themes/cpipr/series-single.php:
```diff
@@ -156,23 +159,22 @@ function getNextPost($post, $series_map)
                   </div>
                 </div>
-              </div>
+							</div>	
```
What happened to the indentation on this line?

rocaiguina replied:
we are now showing the whole block only if $next_post is not empty

benlk commented on wp-content/themes/cpipr/series-single.php:
```diff
@@ -193,6 +195,7 @@ function getNextPost($post, $series_map)
 			</div>
 		</div>
 	</div>
+	<?php endif;?>
```
Either this should be deindented one tab, or the opening end of the `if` statement, and all lines within the `if` statement, should be indented one tab.

rocaiguina replied:
ok, done. please check

benlk commented on wp-content/themes/cpipr/series-single.php:
```diff
@@ -134,19 +134,22 @@ function getNextPost($post, $series_map)
+									<div class="section-title">
+										<h3><?php _e('Next in Series', 'largo'); ?></h3>
+										<?php echo '<h3>' . ($next_position + 1) . ' / ' . count($series_map) . '</h3>'; ?>
+									</div>
```
This section should be indented to match the surrounding code.

rocaiguina replied:
ok, done. please check

kaylima commented on wp-content/themes/cpipr/css/style.css:
```diff
@@ -1535,6 +1537,9 @@ blockquote p {
   .wrapper-special .container-special > .col-xs-6 {
     width: 100%;
   }
+  .wrapper-post-serie {
```
@benlk should this be `.wrapper-post-series` rather than `.wrapper-post-serie`?

benlk replied:
https://github.com/INN/umbrella-cpipr/pull/27/files#diff-491875b64a837adea64560423c9819bcR155 has the class name as `wrapper-post-serie`, so this is correct.

benlk reviewed (changes requested):
Could you make the pagination changes and indentation changes described here? I'm passing this pull request to @kaylima to review the styles.

rocaiguina committed:
Indentation changes

rocaiguina committed:
Reactivated global-nav

rocaiguina commented:
@kaylima @benlk I have applied the requested changes, please let me know if we need to change anything else.

benlk committed:
Make the new for-series-posts single template appear in the template list

Also:
- adds escapes and sanitization
- adds a comment explaining some off-by-one math

benlk committed:
indent matching

benlk committed:
Turns out that WordPress 4.7 introduced post type templates, duplicating Largo's existing post type template work. So I'm partially reverting 89bef22 so that this PR doesn't mess with the existing template choices made in production, and so that it doesn't show up in the Largo box.

benlk commented:
For some reason, I didn't know that https://make.wordpress.org/core/2016/11/03/post-type-templates-in-4-7/ was a thing. That's pretty cool. It's something that we had built out in Largo eons ago (probably 3 years ago with version 0.4), but it looks like WordPress Core added a similar functionality in 2016. Your `series-single.php` uses the WordPress Core method of changing templates, which in the post editor is controlled by the "Post Attributes/Atributos de entrada" meta box in this screenshot:

<img width="312" alt="screen shot 2018-04-25 at 12 26 16 pm" src="https://user-images.githubusercontent.com/1754187/39259263-fb99d4b2-4883-11e8-9b9c-96235cee2a62.png">

If you want to set the template in the Largo way, using the "Layout Options/Opciones de emplanaje" meta box, 89bef22 is the commit that contains those changes. Making that change involves renaming the file, which will reset the layouts on your existing published posts. At some point we'll migrate the Largo way to the WordPress way, but that's not something we need to address right now.

The rest of this looks good, so I'm merging and deploying.

benlk commented:
Now deployed.
